app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/components/return_banner.scala.html

```diff
@@ -16,21 +16,33 @@
 
 @import uk.gov.hmrc.cgtpropertydisposalsfrontend.models.finance.AmountInPence
 @import uk.gov.hmrc.cgtpropertydisposalsfrontend.models.finance.MoneyUtils
+@import uk.gov.hmrc.cgtpropertydisposalsfrontend.models.UserType
+@import uk.gov.hmrc.cgtpropertydisposalsfrontend.models.onboarding.SubscribedDetails
+@import uk.gov.hmrc.cgtpropertydisposalsfrontend.models.returns.ReturnSummary
 
 @this()
 
-@(heading: String, taxOwed: AmountInPence, reference: String)(implicit messages: Messages)
+@(heading: String, returnSummary: ReturnSummary, subscribedDetails: SubscribedDetails, userType: Option[UserType])(implicit messages: Messages)
+
+@taxOwed = @{returnSummary.mainReturnChargeAmount.withFloorZero}
+@name = @{subscribedDetails.name.fold(_.value, e => e.firstName + " " + e.lastName)}
```

Comment:
not sure if you should use this or the `contactName` field - best to double check

Reply:
in this context it is correct to use `name` - it's to show agents and trust the name of the account (not the contact name) the return was posted against